chado/migrations/V1.3.3.005__add_type_id_2_project_table.sql

```diff
@@ -0,0 +1,7 @@
+/* https://github.com/GMOD/Chado/issues/37 */
+ALTER TABLE project
+ADD COLUMN type_id bigint;
+COMMENT ON COLUMN project.type_id IS 'An optional cvterm_id that specifies what type of project this record is.  Prior to 1.4, project type was set with an projectprop.';
+
```

Comment:
I thought you might appreciate some guidance on how to add the requested foreign key and index... i think it should look like this:

```suggestion
ALTER TABLE project ADD FOREIGN KEY (type_id) REFERENCES cvterm (cvterm_id) ON DELETE SET NULL;
CREATE INDEX project_idx1 ON project USING btree (type_id);
```